### PR TITLE
Add dev route with very bare studio playground

### PR DESCRIPTION
**Pull request by paulkaplan**
🙌 Studios work is starting  🙌

It'll be a while until they are ready for primetime, but this adds an initial skeleton of the page, including a new routing system `react-router` to handle tying together the tab navigation and the URL routing. 

It is also the first appearance of react hooks in our codebase :) using the custom hooks from the react-router package to allow easy retrieval of route parameters anywhere in the component tree.

Finally, this PR updates the webpack config to publish routes in the `routes-dev.json` folder (previously just the defunct component library which I'd like to resurrect, now the `/studios-playground/...` route as well) 

/cc @rschamp want to check that it is ok to do that webpack config change for doing dev routes on staging. Seems like it should be ok, just wanted to check. 

By the way the reason I started with the URL routing / tab nav is because I want to default to lazy loading all the non-active tab feeds, just to get an early handle on data requests. 

image

### CODE REVIEW

File: src/routes-dev.json [hunk 1]
```diff
@@ -4,5 +4,12 @@
...
         "title": "Components"
+    },
+    {
+        "name": "studio",
+        "pattern": "^/studios-playground/\\d+(/projects|/curators|/activity|/comments)?/?(\\?.*)?$",
+        "routeAlias": "/studios-playground/?$",
+        "view": "studio/studio",
+        "title": "Studio Playground"
     }
 ]
```
Comment (kchadha) [inline src/routes-dev.json:15]: Missing newline at EOF

Comment (paulkaplan) [reply src/routes-dev.json]: Fixed, thanks

**Comment by paulkaplan** [comment]
@kchadha I removed the part of this that would push the dev routes to staging, pending conversation based on rays feedback offline. We can add it back once we talk about it a bit more.

File: src/views/studio/studio-tab-nav.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                exact
+            >
+                Projects
+            </NavLink>
+            &nbsp;|&nbsp;
+            <NavLink
+                activeStyle={{textDecoration: 'underline'}}
+                to={`${match.url}/curators`}
+            >
+                Curators
```
Comment (kchadha) [inline src/views/studio/studio-tab-nav.jsx:21]: Just noticed that the order of the tabs is different from the current studios page (not that it really matters for this skeleton)

**Review by kchadha — APPROVED** [review]
LGTM

**Pull request merged**